docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,21 @@ NULL
 
 -->
 
+## Version 3.2.3
+
+Date: January 31, 2024
+
+### Improvements
+
+- Added always-restart feature to log-shipping Systemd service, to able to keep it still alive in case OOM-kill happen by the kernel.
+- Added FDS-based log-file discovery as a fallback to eBPF-based log-file discovery.
```

Comment:
FDS => file descriptor

Also, these shound a bit better to me, but feel free to ignore 
```
- Implemented an "always-restart" feature in the log-shipping Systemd service to ensure its continuous operation, even in the event of an OOM-kill triggered by the kernel.
- Enhanced our capability to verify file modified-time values. 
```